File: VT05.py
number_n, number_need_count = map(int, input().split())
number_need_count = abs(int(number_need_count))
number_array = list(map(int,input().split()))
number_array = [abs(x) for x in number_array]
number_dict = {}
for item in number_array:
    if item not in number_dict:
        number_dict[item] = 1
    else:
        number_dict[item] += 1

if number_need_count not in number_dict:
    print(0)
else:
    print(number_dict.get(number_need_count))

File: VT05/VT05.cpp
#include<bits/stdc++.h>
using namespace std;

int main() {
    int number_n, number_need_count;
    cin >> number_n >> number_need_count;
    number_need_count = abs(number_need_count);
    vector<int> number_array(number_n);
    for (int i = 0; i < number_n; i++) {
        cin >> number_array[i];
        number_array[i] = abs(number_array[i]);
    }
    map<int, int> number_dict;
    for (int item : number_array) {
        number_dict[item]++;
    }
    if (number_dict.find(number_need_count) == number_dict.end()) {
        cout << 0 << endl;
    } else {
        cout << number_dict[number_need_count] << endl;
    }
    return 0;
}
